README.md

```diff
@@ -39,10 +39,17 @@ def mystery2(n)
 end
 ```
 
-- What is mystery2(123)?
-- What is mystery2(9005)?
-- What is mystery2(-123)?
+- What is mystery2(123)? 6
+- What is mystery2(9005)? 14
+- What is mystery2(-123)? -123
 - _Added Fun: How could we make `mystery2(-123)` work the way we might expect it to work instead of the way it does?_
+- It looks like this method is created in order to add all the digits together of each number.  Meaning a number entered 123 should result in 6.  To account for negative numbers, we could create an earlier if statement:
+
+if n < 0 && n >= -10
+  return n
+else
+  return (n%10) - mystery2(n/10)
+end
```

Comment:
Hmm... I don't think this is right. What happens if you did mystery2(-12)? I would expect -3, but I don't think that's what your modification would yield.